### PR TITLE
add article and links

### PR DESCRIPTION
**Pull request by Amarant84**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,6 +3,24 @@
 <title> IT memes</title>
 </head>
 <body>
+<h1>Kto może zostać programistą</h1>
+<p><a href="https://www.google.pl/search?sxsrf=ACYBGNRcpMJwaWIj5_8CfBRjhlfqdVH_zg%3A1573915719933&source=hp&ei=RwzQXYDANsqFwPAP_e-t6As&q=programowanie&oq=programowanie&gs_l=psy-ab.3..35i39j0l9.778.2533..3046...0.0..0.105.1066.12j1......0....1..gws-wiz.......0i131.xfIHeQWT76Y&ved=0ahUKEwjA_4_7_O7lAhXKAhAIHf13C70Q4dUDCAU&uact=5" target="_blank">Programowanie</a> to zajęcie, które wymaga znajomości wielu zagadnień. To wiedza, którą zdobywa się przez kilka ładnych lat.
```
Comment (tolhith) [inline index.html]: change to another webpage

File: index.html [hunk 2]
```diff
@@ -3,6 +3,24 @@
...
+
+Nie bez powodu piszę <a href="https://www.google.pl/search?sxsrf=ACYBGNQbGWLMi4YSWTA1Al3U_G3ZoJl6Og%3A1573915782605&ei=hgzQXf3IJJLRrgSUjrCACg&q=blog&oq=blog&gs_l=psy-ab.3..0i67j0l2j0i131l3j0l4.22649.23201..23531...0.2..0.103.362.3j1......0....1..gws-wiz.......0i71j35i39.nsURPl9LMPk&ved=0ahUKEwj9o4OZ_e7lAhWSqIsKHRQHDKAQ4dUDCAo&uact=5" target="_blank">blog</a> po polsku. Zależy mi na tym, żeby ludzie, którzy nie czują się swobodnie czytając <i>angielskie</i> 
+teksty również znaleźli coś dla siebie. Jednak muszę to powiedzieć z pełną stanowczością. Język angielski to podstawa. 
+Jest to uniwersalny język komunikacji, który każdy programista po prostu musi znać. Jeśli go nie znasz zachęcam do 
+przyłożenia się do zajęć z <a href="https://www.google.pl/search?sxsrf=ACYBGNR7A5ICIH6t6ZHaPGxZVH7vyzB9hg%3A1573915723369&ei=SwzQXeuZFp-EwPAPt6uG8Aw&q=j%C4%99zyk+angielski&oq=j%C4%99zyk+angi&gs_l=psy-ab.1.0.0l10.146270.148460..150317...0.2..0.96.791.10......0....1..gws-wiz.......0i71j35i39j0i67j0i131.nTWiL6RvV64" target="_blank">języka angielskiego</a>. Jeśli nie masz dostępu do takich zajęć w internecie można znaleźć bardzo dużo darmowych materiałów pomagających w nauce.
+
```
Comment (tolhith) [inline index.html:19]: insert <br><br><br>

File: index.html [hunk 3]
```diff
@@ -3,6 +3,24 @@
...
+<p><a href="https://www.google.pl/search?sxsrf=ACYBGNRcpMJwaWIj5_8CfBRjhlfqdVH_zg%3A1573915719933&source=hp&ei=RwzQXYDANsqFwPAP_e-t6As&q=programowanie&oq=programowanie&gs_l=psy-ab.3..35i39j0l9.778.2533..3046...0.0..0.105.1066.12j1......0....1..gws-wiz.......0i131.xfIHeQWT76Y&ved=0ahUKEwjA_4_7_O7lAhXKAhAIHf13C70Q4dUDCAU&uact=5" target="_blank">Programowanie</a> to zajęcie, które wymaga znajomości wielu zagadnień. To wiedza, którą zdobywa się przez kilka ładnych lat. 
+Nie znam żadnego <b>programisty</b>, który nauczył się wszystkiego w jeden wieczór. To jest po prostu niemożliwe. Tak i na tym blogu 
+będziemy razem przechodzili przez serię artykułów, które pomogą Ci zgłębić podstawowe zagadnienia. Pokażę Ci dalszą drogę, która pomoże Ci zostać programistą.
 
+<i>Programowanie</i> jako zajęcie wymaga pewnych umiejętności. <b>Umiejętności</b> te są pomocne przy zgłębianiu niezbędnej wiedzy. 
+Poza umiejętnościami bardzo istotne jest także zaangażowanie i wytrwałość. Oczywiście “talent” pomaga, jednak ciężka, 
+systematyczna praca to podstawa. Bez talentu można programować, jednak bez ciężkiej pracy włożonej w naukę już nie (jestem tego bardzo dobrym przykładem).
+
```
Comment (tolhith) [inline index.html:14]: insert <br><br><br>

File: index.html [hunk 4]
```diff
@@ -3,6 +3,24 @@
...
+Nie bez powodu piszę <a href="https://www.google.pl/search?sxsrf=ACYBGNQbGWLMi4YSWTA1Al3U_G3ZoJl6Og%3A1573915782605&ei=hgzQXf3IJJLRrgSUjrCACg&q=blog&oq=blog&gs_l=psy-ab.3..0i67j0l2j0i131l3j0l4.22649.23201..23531...0.2..0.103.362.3j1......0....1..gws-wiz.......0i71j35i39.nsURPl9LMPk&ved=0ahUKEwj9o4OZ_e7lAhWSqIsKHRQHDKAQ4dUDCAo&uact=5" target="_blank">blog</a> po polsku. Zależy mi na tym, żeby ludzie, którzy nie czują się swobodnie czytając <i>angielskie</i> 
+teksty również znaleźli coś dla siebie. Jednak muszę to powiedzieć z pełną stanowczością. Język angielski to podstawa. 
+Jest to uniwersalny język komunikacji, który każdy programista po prostu musi znać. Jeśli go nie znasz zachęcam do 
+przyłożenia się do zajęć z <a href="https://www.google.pl/search?sxsrf=ACYBGNR7A5ICIH6t6ZHaPGxZVH7vyzB9hg%3A1573915723369&ei=SwzQXeuZFp-EwPAPt6uG8Aw&q=j%C4%99zyk+angielski&oq=j%C4%99zyk+angi&gs_l=psy-ab.1.0.0l10.146270.148460..150317...0.2..0.96.791.10......0....1..gws-wiz.......0i71j35i39j0i67j0i131.nTWiL6RvV64" target="_blank">języka angielskiego</a>. Jeśli nie masz dostępu do takich zajęć w internecie można znaleźć bardzo dużo darmowych materiałów pomagających w nauce.
```
Comment (tolhith) [inline index.html]: can it open in embedded frame?

**Review by tolhith — CHANGES_REQUESTED** [review]
Thank you for your contribution. Please address my comments. Thanks!

**Review by Przemo75 — COMMENTED** [review]
Rozbij, proszę, tekst na na kilka akapitów

**Comment by Amarant84** [comment]
fixes #3, fixes #4, fixes #6, fixes #5

**Review by tolhith — COMMENTED** [review]
Fixes #2

**Review by Przemo75 — COMMENTED** [review]
OK :-)

**Pull request merged**